<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1051</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="generate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Generate settings</string>
     </attribute>
     <widget class="QPushButton" name="add_prod">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add product</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="set_days">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="value">
       <number>365</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Shops</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="set_shops">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Days</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_gen">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Start simulation</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>1031</width>
        <height>441</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="next_days_but2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Next day</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gen_all_but2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Simulate all days</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="storage">
     <attribute name="title">
      <string>Storage</string>
     </attribute>
     <widget class="QTableWidget" name="table_storage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1031</width>
        <height>491</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="next_day_but">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>500</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Next day</string>
      </property>
     </widget>
     <widget class="QPushButton" name="all_days_but">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>500</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Simulate all days</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Factory</string>
     </attribute>
     <widget class="QTableWidget" name="table_factory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1031</width>
        <height>491</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="next_day_but3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>500</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Next day</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gen_all_but3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>500</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Simulate all days</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Stats</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Profit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lost money:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="profit_text">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lost_text">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Trashed products:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="trashed_text">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Discounted products sold:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="discounted_text">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Days passed:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passed">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
